### PR TITLE
[IOTDB-2650] Tablet supports adding String value

### PR DESCRIPTION
**Pull request by jiazhiren**
This PR creates syntactic sugar by supporting String value insertion to Tablet. #[IOTDB-2650]

### CODE REVIEW

File: tsfile/src/main/java/org/apache/iotdb/tsfile/write/record/Tablet.java [hunk 1]
```diff
@@ -138,7 +138,7 @@ private void addValueOfDataType(
       case TEXT:
         {
           Binary[] sensor = (Binary[]) values[indexOfSchema];
-          sensor[rowIndex] = value != null ? (Binary) value : Binary.EMPTY_VALUE;
+          sensor[rowIndex] = value != null ? new Binary((String) value) : Binary.EMPTY_VALUE;
```
Comment (HTHou) [inline tsfile/src/main/java/org/apache/iotdb/tsfile/write/record/Tablet.java:141]: Thanks for your contribution!

IMO, we should also support adding a Binary value here to keep the compatibility. :)

Comment (HTHou) [reply tsfile/src/main/java/org/apache/iotdb/tsfile/write/record/Tablet.java:141]: Besides, consider adding some tests in `IoTDBSessionSimpleIT`?

Comment (jiazhiren) [reply tsfile/src/main/java/org/apache/iotdb/tsfile/write/record/Tablet.java:141]: Thanks for your valuable advice! I will work on them immediately.

**Comment by HTHou** [comment]
Hi, you can execute `mvn spotless:apply` to fix the test issue.

**Comment by jiazhiren** [comment]
> 

Thanks for your tip. The code format has been corrected.

**Review by HTHou — APPROVED** [review]

**Pull request merged**